rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/notifications/OpenUnityProjectAsFolderNotification.kt

```diff
@@ -70,11 +80,6 @@ class OpenUnityProjectAsFolderNotification(project: Project) : ProtocolSubscribe
                         ProjectManagerEx.getInstanceEx().closeAndDispose(project)
                         WelcomeFrame.showIfNoProjectOpened()
                     }
-                    if (hyperlinkEvent.description == "reopen") {
-                        StartUnityAction.startUnityAndRider(project)
-                        ProjectManagerEx.getInstanceEx().closeAndDispose(project)
-                        WelcomeFrame.showIfNoProjectOpened()
-                    }
```

Comment:
Why remove this?

Reply:
I don't like how it works, it closes folder and may take a very long time to open Unity with project and only if Rider package is there and Rider is set as extenral Editor it would start it. May be doing more harm then good.